README.md

```diff
@@ -35,13 +35,14 @@ Where additional sophistication with AVSs come into the picture:
 ### Steps
 
 1. Make sure Docker is running
-2. Run `make start-chain-with-contracts-deployed`
+2. Run `npm install`
+3. Run `make start-chain-with-contracts-deployed`
 
     2.1 This will build the contracts, start an Anvil chain, deploy the contracts to it, and leaves the chain running in the current terminal
```

Comment:
```suggestion
    3.1 This will build the contracts, start an Anvil chain, deploy the contracts to it, and leaves the chain running in the current terminal
```
```suggestion
    2.1 This will build the contracts, start an Anvil chain, deploy the contracts to it, and leaves the chain running in the current terminal
```
```suggestion
    2.1 This will build the contracts, start an Anvil chain, deploy the contracts to it, and leaves the chain running in the current terminal
```